client/racks.go

```diff
@@ -0,0 +1,17 @@
+package client
+
+type Organization struct {
+	Id   string `json:"id"`
+	Name string `json:"name"`
+}
+
+type Rack struct {
```

Comment:
We also have a struct that covers this in client.System. But I think that one is poorly named, so re-introducing a Rack struct is good.